lib/user_agent/browsers/stagefright.rb

```diff
@@ -49,7 +49,7 @@ def application
       #
       # @return [String]
       def platform
-        'Android' if ANDROID_REGEX.match?(self.to_s)
+        'Android'
```

Comment:
freeze or use pragma?

Reply:
After looking at this again, I think that we need to detect Fire OS separately from Android (since it's an incompatible fork of Android). If we're not comfortable making this choice at the moment, then this is probably an OK change.. though I am curious what instances of stagefright weren't coming in as Android